md-commons/src/main/java/com/wipro/ats/bdre/md/beans/table/AppDeploymentQueueStatus.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.wipro.ats.bdre.md.beans.table;
+
+/**
+ * Created by SU324335 on 3/8/2016.
+ */
+public class AppDeploymentQueueStatus {
+    private Short appDeploymentStatusId;
+    private String description;
+    private Integer pageSize;
+    private Integer page;
+    private Integer counter;
+
+    public AppDeploymentQueueStatus() {
+    }
+
+    public AppDeploymentQueueStatus(Short adqState, String description) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused method parameter(s) "adqState". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1172)